docs/book/src/tasks/certs/auto-rotate-certificates-in-kcp.md

```diff
@@ -0,0 +1,64 @@
+## Automatically rotating certificates using Kubeadm Control Plane provider
+
+When using Kubeadm Control Plane provider (KCP) it is possible to configure automatic certificate rotations. KCP does this by triggering a rollout when the certificates on the machines are about to expire.
```

Comment:
```suggestion
When using Kubeadm Control Plane provider (KCP) it is possible to configure automatic certificate rotations. KCP does this by triggering a rollout when the certificates on the control plane machines are about to expire.
```
I know it can be inferred but maybe good to emphasize